src/Tasks/Microsoft.NET.Build.Tasks/NETSdkError.cs

```diff
@@ -24,13 +24,22 @@ public class NETSdkError : TaskBase
         /// </summary>
         public string[] FormatArguments { get; set; }
 
+        public bool WarningOnly { get; set; }
```

Comment:
I would prefer NETSdkWarning and NETSdkError tasks just like the built-in Error and Warning tasks. It is weird to read Error then Warning only.